fenix/app/src/main/java/org/mozilla/fenix/gleanplumb/MessageNotificationWorker.kt

```diff
@@ -14,14 +14,14 @@ import android.os.Bundle
 import android.os.IBinder
 import androidx.core.app.NotificationManagerCompat
 import androidx.work.ExistingPeriodicWorkPolicy
-import androidx.work.PeriodicWorkRequest
+import androidx.work.PeriodicWorkRequestBuilder
 import androidx.work.WorkManager
 import androidx.work.Worker
 import androidx.work.WorkerParameters
 import kotlinx.coroutines.DelicateCoroutinesApi
-import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
```

Comment:
Detekt suggests we should be using `runBlockingIncrement` here.

```suggestion
import org.mozilla.fenix.perf.runBlockingIncrement as runBlocking
```